core/src/main/scala/org/apache/spark/scheduler/cluster/mesos/MesosCoarseGrainedSchedulerBackend.scala

```diff
@@ -423,18 +432,35 @@ private[spark] class MesosCoarseGrainedSchedulerBackend(
     tasks.toMap
   }
 
+  private def getResources(resources: JList[Resource], taskCPUs: Int, taskMemory: Int)
```

Comment:
Since this is partitioning resources for the task, can we call this `partitionTaskResources`?  `getResources` is vague